src/components/TransactionHistoryRows/TransactionGenericRow.tsx

```diff
@@ -116,6 +107,47 @@ export const TransactionGenericRow = ({
     breakPoints: [isLargerThanLg],
   } = GetTxLayoutFormats({ parentWidth })
 
+  const DisplayTransfers = useMemo(
```

Comment:
Nitpick, but `displayTransfers` for the memoized node would be conventionally more fit, as uppercase is conventionally used for components, and this is a memoized JSX node